script.js

```diff
@@ -1,4 +1,7 @@
 /* eslint-disable no-plusplus */
```

Comment:
- [ ] **Required:** Kindly fix linter errors, rather than disable them as they ensure a uniform codebase in the community.

Reply:
I think we have the freedom to use this. Because it seems like a bug.